spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -240,17 +215,16 @@
 """
 
 
-def getHeader(productName, options, orgCACert, isRpmYN, pubname, apachePubDirectory):
+def getHeader(productName, options, orgCACert, pubname, apachePubDirectory):
     # 11/22/16 options.gpg_key is now a comma-separated list of path.
     # Removing paths from options.gpg_key
     org_gpg_key = ",".join([os.path.basename(gpg_key) for gpg_key in options.gpg_key.split(",")])
-    exit_call = " " if options.activation_keys or options.salt else "exit 1"
     with cfg_component('web') as CFG:
         version = CFG.version
-        if isUyuni:
+        if isUyuni():
```

Comment:
IMO this could be an attribute instead of a function, but that would require changing all existing consumers.

Reply:
Right. And I just found out that the current version does not work. So we just leave it for now.